### ISSUE TITLE
Redirect Url items and Langauges

### ISSUE THREAD

**Issue by basselissa** [thecadams/301RedirectModule#51]
I was wondering if we can have the request url field unversioned 
does that allow us use the same item for other langauges?
like 
/welcome   -->   /en-us/weclomepage    for the English version
/welcomen --> /de-du/welcomen    for the Dutch version

**Comment by thecadams** [comment]
Absolutely, although there might be code changes to make sure the right item is used for the user making the request.

You’d be very welcome to put whatever is needed into a PR. Unfortunately I am not actively working on the module any more but can happily merge your change, if we can get it reviewed by another member of the project.

**Comment by basselissa** [comment]
will see what i can share soon.
